### PR TITLE
Emote volume channel + volume QoL

### PR DESCRIPTION
**Pull request by pali6**
[qol] [input wanted] [cleanliness]
## About the PR 

Adds a new volume channel - screamfart. This channel contains most screams and farts which are known to annoy a lot of users, now you can change volume of these sounds separately.
Also reorganizes volume settings, now they are all in the audio tab:
image
Also adds unlisted `change-volume` verb that you can use if you are too lazy to click the Audio menu thingie.

### **EDIT: DUE TO INTERNAL FEEDBACK CHANGED TO INCLUDE ALMOST ALL EMOTE SOUNDS**
Channel is now named the Emote Channel and includes all emotes with the following exceptions:
*deathgasp, *gasp due to their gameplay importance
flock sounds due to their possible importance in the future if flock gamemode ever becomes a regular thing

## Why's this needed? 
A fair share of players find scream and fart sounds annoying, especially if they happen constantly (and I don't blame them). Allowing for separate adjustment of their volume should help.

## Changelog
```
(u)pali
(*)You can now adjust volume of emotes in the audio tab in the top bar.
```

### CODE REVIEW

**Comment by UrsulaMejor** [comment]
give farts and screams overhead text like salute and wave do and you have my vote

**Comment by Ley-bunned** [comment]
do it *without* overhead text and you've got my vote, because people spamming it is awful, annoying and the number 1 thing that stops me from telling friends what SS13 server I play on

**Comment by UrsulaMejor** [comment]
If that can't even abide the presence of the words "fart" "or "scream" appearing on their screen, they prolly wouldn't enjoy a game where we have an entire adventure zone dedicated to farts and where farting on people is a regular greeting.

I can understand why the sound can be annoying, but if the text is really that much of an issue for you, the text is still going to appear in the chat log anyways. At that point you're not asking for a sound toggle, you're asking for a toggle to remove farting as a system

**Comment by TTerc** [comment]
we can always go for the ultimate compromise and just make a toggle for screamfarts in floating chat

**Comment by Flaborized** [comment]
IMO overhead text would be annoying for farting, which people do constantly and subconsciously. Farting has a very minor mechanical effect on the game, the ability to disable the audio if you want doesn't change anything major other than disabling a (admittedly pretty annoying) sound. You could argue that screaming loses some ability to get attention with this change, but *nobody screams to get people's attention*, they just do it passively all the time. 

Maybe a message for... farting in someone's face would be sufficient. It would suck to know that people might not even notice it if you pull it off, but for other fart sounds I think the sounds themselves add little enough that I wouldn't care if people were allowed to turn them off.

**Comment by TTerc** [comment]
Throwing this idea from discord here: exclude high fives and slaps from the volume channel since they're more difficult to spam anyway. Wendingo and other violent critter screams should also be excluded imo

**Comment by pali6** [comment]
Wendigos and other critters are objcritters which this PR doesn't touch. Mobcritter wendigos and other violent critters exist but are generally only admin-spawnable and have no AI.

**Comment by Carbadox** [comment]
what does emote sounds even encompass? the only ones i can recall are highfives, slaps and burps, outside of the ones already mentioned. If there's a need for a volume control for these then, i dont get it but sure. I still think farting deserves its own seperate channel though

File: code/modules/sound/sound.dm [hunk 1]
```diff
@@ -59,18 +59,24 @@ var/global/ECHO_AFAR = list(0,0,0,0,0,0,-10000,1.0,1.5,1.0,0,1.0,0,0,0,0,1.0,7)
 var/global/ECHO_CLOSE = list(0,0,0,0,0,0,0,0.25,1.5,1.0,0,1.0,0,0,0,0,1.0,7)
 var/global/list/falloff_cache = list()
 
+//default volumes
+var/global/list/default_channel_volumes = list(1, 1, 0.1, 0.5, 0.5, 1)
 
 //volumous hair with l'orial paris
-/client/var/list/volumes = list(1, 1, 0.1, 0.5, 0.5)
+/client/var/list/volumes = list(1, 1, 0.1, 0.5, 0.5, 1)
```
Comment (ZeWaka) [inline code/modules/sound/sound.dm:66]: why not default_channel_volumes.Copy()

**Review by ZeWaka — APPROVED** [review]

**Pull request merged**